src/main/java/Event.java

```diff
@@ -0,0 +1,22 @@
+public class Event extends Todo{
+    protected String start;
+    protected String end;
+    public Event(String description, String start, String end)
+    {
+        super(description);
+        this.start = start;
+        this.end = end;
+    }
+    public String getTypeIcon() {
+        return ("[E]"); // mark done task with X
```

Comment:
I think this comment might be wrong (or duplicated)

Reply:
thank you for pointing it out